src/pymedphys/cli/main.py

```diff
@@ -27,7 +27,9 @@
 import json
 import argparse
 
-from ..dicom import adjust_machine_name_cli, adjust_rel_elec_density_cli
+from ..dicom import (
+    adjust_machine_name_cli, adjust_rel_elec_density_cli,
+    structure_name_RED_adjust_cli)
```

Comment:
This is an awkward name. Maybe `adjust_RED_by_structure_name_cli`? Also, you've used "RED" here, but "rel_elec_density" in `adjust_rel_elec_density_cli`. Best to keep consistent?

Reply:
Yup I agree, those a far better